io.github.zer0x.qurancompanion.yml

```diff
@@ -0,0 +1,43 @@
+app-id: io.github.zer0x.qurancompanion
```

Comment:
`zer0x` isn't the org/user name on github. The app-id doesn't meet the requirements.

Reply:
error: 'io.github.0xzer0x.qurancompanion' is not a valid application name: Name segment can't start with 0

Reply:
What are valid ids I could use?

Reply:
https://dbus.freedesktop.org/doc/dbus-specification.html#message-protocol-names

Linked from https://docs.flatpak.org/en/latest/conventions.html#application-ids

put a `_` before the initial `0`

Reply:
thanks, done.